Userland/Libraries/LibAudio/Sample.h

```diff
@@ -45,6 +46,12 @@ struct Sample {
     constexpr Sample(Sample&& other) = default;
     constexpr Sample& operator=(Sample&& other) = default;
 
+    // *Either* channel has a NaN value.
+    bool is_nan() const
```

Comment:
If you have to add a comment to explain what a function does, maybe the function could explain itself better by its name?

E.g. `either_channel_is_nan()` or `any_channel_is_nan()`